xml/System.Web.Security/Membership.xml

```diff
@@ -116,7 +116,7 @@
    
   
 ## Examples  
- The following code example shows the  element in the `system.web` section of the Web.config file for an ASP.NET application. It specifies that the application use a <xref:System.Web.Security.SqlMembershipProvider> instance and sets the <xref:System.Web.Security.Membership.ApplicationName%2A> to `MyApplication`.  
+ The following code example shows the `<membership>` element in the `system.web` section of the Web.config file for an ASP.NET application. It specifies that the application use a <xref:System.Web.Security.SqlMembershipProvider> instance and sets the <xref:System.Web.Security.Membership.ApplicationName%2A> to `MyApplication`.  
```

Comment:
membership element: <https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/1b9hw62f(v%3dvs.100)> (also on lines 532, 590, 1426, 1468, 1475, 1530, 1577, 1582, 1635, 1642, 1680, 1696, 1787, 1864, and 1970)